packages/esm-patient-banner-app/src/print-identifier-sticker/print-identifier-sticker.component.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { forwardRef, useMemo } from 'react';
```

Comment:
```suggestion
import React from 'react';
```

Please remove the unused imports